mara_pipelines/logging/run_log.py

```diff
@@ -103,6 +110,13 @@ class RunLogger(events.EventHandler):
     node_output: {tuple: [pipeline_events.Output]} = None
 
     def handle_event(self, event: events.Event):
+        import mara_db.config
+
+        if 'mara' not in mara_db.config.databases():
+            return
```

Comment:
It feels slightly wrong to me to do this here instead of the place where this RunLogger is getting instantiated. But well, it is what it is...

(if this should get refactored, the system statistics should probably get it's own handler. Or remove it, as probably not a lot of ppl run it locally on a pg anymore and thats the only case where the stats are relevant anymore: not docker, not cloud :-()

Reply:
I agree that this was a dirty solution. I changed this now that `RunLogger` is a base class without any functionality and a new class `DbRunLogger` has the db insert logic. A warning is added when `RunLogger` is used because no logging of events takes place.

This introduces now a breaking change when someone would actively use the `RunLogger` class, but makes the whole logic more generic. I thought already implementing a kafka run logger for pipeline events ...

Related to the system statstics: 100% agree. I don't use it either. Maybe should be handled in another ticket. I think it would be nice to have a config option to disable it. This config should then hide the system statistics from the UI as well.

Reply:
see below